From 70dde6f00b6c8b5a516cc7d251bc27dc374ee28f Mon Sep 17 00:00:00 2001
From: Nicolas Gordillo <nicolas.gordillo@swisscom.com>
Date: Sat, 6 Apr 2019 11:20:01 +0200
Subject: [PATCH] q

---
 insertcoffdetails.php | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/insertcoffdetails.php b/insertcoffdetails.php
index 3d524a4..2bce151 100644
--- a/insertcoffdetails.php
+++ b/insertcoffdetails.php
@@ -1,6 +1,6 @@
 <?php
 
-  if(isset($_POST['send'])){
+  if(!isset($_POST['send'])){
    $link = mysqli_connect("localhost", "root", "K98awakn", "coffee");
 
     // Check connection
-- 
2.20.1

